lib/utils/messages.ts

```diff
@@ -1,15 +1,22 @@
+const DEBUG_MODE = !!process.env.DEBUG;
+
 export const reachingOut = `\
 If you need help, or just want to say hi, don't hesitate in reaching out at:
 
-  GitHub: https://github.com/balena-io/balena-cli/issues/new
-  Forums: https://forums.balena.io\
+  Forums: https://forums.balena.io
+  GitHub: https://github.com/balena-io/balena-cli/issues/new\
+`;
+
+const debugHint = `\
+Additional information may be available in debug mode by prefixing the
```

Comment:
How about ?
```suggestion
Additional information may be available if you run the CLI in debug mode. To do so prefix the
```

Reply:
Thanks Thodoris. I am not a fan of using _'you'_ in help messages, but I've reworded it to avoid _'by prefixing'_ which I think is the bit you wanted to avoid.